gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -109,6 +109,8 @@
 
 #define g_free_const(x) g_free((void*)(long)(x))
 
+#define REGEX_NNAPI_OPTION "((true|false)$|(true|false)([:]?(cpu|gpu|npu)))"
```

Comment:
I am not sure but
```suggestion
#define REGEX_NNAPI_OPTION "(^(true|false)$|^(true|false)([:]?(cpu|gpu|npu)))"
```
could be more exact rule for 'BOOL:ACCELLERATOR'.

Q. true (without any ACCELERATOR) could be passed by this regex. Is it your intention? If so, you need to add BOOL as the format of the option string in addition to BOOL:ACCELERATOR at the line 764.

Reply:
Right. It's my intention. If it is just true, the use cpu as default. I'll add BOOL at the line 764.